src/deepforest/utilities.py

```diff
@@ -354,8 +354,8 @@ def read_file(input, root_dir=None):
             raise ValueError("No annotations in dataframe")
         # If the geometry column is present, convert to geodataframe directly
         if "geometry" in df.columns:
-            df['geometry'] = gpd.GeoSeries.from_wkt(df['geometry'])
-            df.crs = None
+            if all(isinstance(x, str) for x in df['geometry']):
```

Comment:
Is there a way we can solve this issue with out specifically looping through the data frame?

Reply:
Use the option you feel works best and less complexity than looping